packages/html/src/multiselecttree/tests/multiselecttree-size.tsx

```diff
@@ -13,38 +13,20 @@ export default () =>(
     <>
         <style>{styles}</style>
         <div id="test-area" className="k-d-grid k-grid-cols-3">
-            <span>small</span>
-            <MultiSelectTree
-                size="small"
-                tags={(
-                    <>
-                        <Chip text="Chip Text" size="small" actions={ <ChipAction type="remove"/> } />
-                        <Chip text="Chip Text" size="small" actions={ <ChipAction type="remove"/> } />
-                    </>
-                )}
-            />
-
-            <span>medium</span>
-            <MultiSelectTree
-                size="medium"
-                tags={(
-                    <>
-                        <Chip text="Chip Text" size="medium" actions={ <ChipAction type="remove"/> } />
-                        <Chip text="Chip Text" size="medium" actions={ <ChipAction type="remove"/> } />
-                    </>
-                )}
-            />
-
-            <span>large</span>
-            <MultiSelectTree
-                size="large"
-                tags={(
-                    <>
-                        <Chip text="Chip Text" size="large" actions={ <ChipAction type="remove"/> } />
-                        <Chip text="Chip Text" size="large" actions={ <ChipAction type="remove"/> } />
-                    </>
-                )}
-            />
+            {MultiSelectTree.options.size.map((size) => (
+                <>
+                    <span>{size}</span>
+                    <MultiSelectTreeNormal
```

Comment:
You should handle the rounded options here as well. Check this test for reference: https://github.com/telerik/kendo-themes/blob/develop/packages/html/src/multiselect/tests/multiselect-size-rounded.tsx